test/csharp/edge/Main.cs

```diff
@@ -110,12 +131,16 @@ public ProviderStack(Construct scope, string id) : base(scope, id)
                 Alias = "full"
             });
 
-            // Not supported .ReqXXX is optional in CSharp
-            // new RequiredAttributeResource(this, "reqOpt", new RequiredAttributeResourceConfig {
-            //     Bool = providerOpt.Reqbool,
-            //     Num = providerOpt.Reqnum,
-            //     Str = providerOpt.Reqstr
-            // });
+            // TODO: this currently does not compile because provider.reqbool may be undefined
```

Comment:
Is this TODO still up to date?

Reply:
Was copied from the TS test. I'll update these to have a note, but drop the TODO.